book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -1,10 +1,34 @@
 (rr-licensing-ml)=
-# Machine Learning Model Licenses
+# Licensing Machine Learning models
+
+## Legal Status of AI/ML model weights
+
+It is an open question wether AI/ML models weights are even copyrightable.
+US copyright law specifically excludes the following from works eligible for copyright protection: "any idea, procedure, process, system, method of operation, concept, principle, or discovery, regardless of the form in which it is described, explained, illustrated, or embodied in such work."
+In addition the US Copyright Office has stated that this exclusion extends to "scientific or technical methods or discoveries;" "mathematical principles;" and "formulas or algorithms."
+
+A legal doctrine known as the idea–expression distinction/dichotomy draws a distinction between an abstract idea and a specific implementation, where code authored by a human that implements an algorithm is copyrightable as a creative work of authorship but the algorithm itself is not.
+
+<!--see: https://www.comsol.com/blogs/can-models-be-protected-by-copyright-law/-->
+<!--see: https://en.wikipedia.org/wiki/Idea%E2%80%93expression_distinction -->
```

Comment:
Do we want to add those items as links in the paragraph?

Reply:
Addressed in [325edf8](https://github.com/the-turing-way/the-turing-way/pull/3142/commits/325edf845bb0d5d71807dd44f32ca9ccb412156c)